packages/backend-for-frontend/src/services/privacyNoticeService.ts

```diff
@@ -134,19 +134,18 @@ export function privacyNoticeServiceBuilder(
         `Retrieving privacy notice content for consentType ${consentType}`
       );
 
-      const privacyNoticeId = retrievePrivacyNoticeId(
-        consentType,
-        consentTypeMap
-      );
-
-      const latest = await retrieveLatestPrivacyNoticeVersion(
-        consentType,
-        privacyNoticeId,
-        privacyNoticeStorage,
-        logger
-      );
-
-      const path = `${config.privacyNoticesPath}/${privacyNoticeId}/${latest.privacyNoticeVersion.versionId}/it/${config.privacyNoticesFileName}`;
+      const path = match(consentType)
+        .with(
+          "PP",
+          () =>
+            `${config.privacyNoticesPath}/latest/it/${config.privacyNoticesPPFileName}`
+        )
+        .with(
+          "TOS",
+          () =>
+            `${config.privacyNoticesPath}/latest/it/${config.privacyNoticesTOSFileName}`
```

Comment:
Could you define the base path `${config.privacyNoticesPath}/latest/it/` in a common variable?
Also, could you update the locations of the files and the filenames in `docker/minio-seed/` so that you can test the calls in the local env 🙏